issue-003/solution.js

```diff
@@ -1,3 +1,12 @@
 function product(a, b) {
     // implement your solution here
+    try {
```

Comment:
I don't think the `try..catch` block was needed here. You only needed to _throw_ the error. The person calling the function is the one that will need the `try..catch` block.